.github/cmake/github_ci.cmake

```diff
@@ -0,0 +1,47 @@
+if (NOT DEFINED "ENV{CI}")
+  message(FATAL_ERROR "This script assumes it is run inside a Github CI action.")
+endif()
+
+# Set the build metadata.
+set(CTEST_BUILD_NAME "$ENV{GITHUB_REPOSITORY}-$ENV{CMAKE_CONFIGURATION}")
+set(CTEST_SITE "gitlab-ci")
```

Comment:
```suggestion
set(CTEST_SITE "github-ci")
```

Reply:
Oops. You can see a lot of this is a copy from our gitlab CI workflow. Fixed.